client/packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/StocktakeLineEditTables.tsx

```diff
@@ -216,6 +215,17 @@ export const BatchTable = ({
         accessor: ({ rowData }) =>
           rowData.packSize ?? rowData.item?.defaultPackSize,
         defaultHideOnMobile: true,
+        setter: patch => {
+          const shouldClearSellPrice =
+            patch.item?.defaultPackSize !== patch.packSize &&
+            patch.item?.itemStoreProperties?.defaultSellPricePerPack ===
+              patch.sellPricePerPack;
+          if (shouldClearSellPrice) {
+            update({ ...patch, sellPricePerPack: 0 });
+          } else {
+            update(patch);
```

Comment:
Doesn't work for me when using + batch button, then changing the pack size 😢  I think itemStoreProperties isn't available on the item?

Reply:
Weird... all seems to be working for me 👀 

https://github.com/user-attachments/assets/f10511e3-3882-48e3-98b1-f63d2faf31fe